generators/generator-eclipseditto/src/main/java/org/eclipse/vorto/codegen/ditto/schema/tasks/template/DittoStructureTemplate.xtend

```diff
@@ -30,13 +30,13 @@ class DittoStructureTemplate implements IFileTemplate<InformationModel> {
 	override getPath(InformationModel context) {
 		return null;
 	}
-	
+
 	override getContent(InformationModel model, InvocationContext context) {
 		'''
 {
+  "definition": ["«model.namespace»:«model.name»:«model.version»"],
```

Comment:
Hi @kolotu - this is not exactly correct.
The "definition" on Thing level is not a Json array, but a Json string, see also the documentation: https://www.eclipse.org/ditto/basic-thing.html#api-version-2

Reply:
Hi @thjaeckle - thanks for pointing that out, I'll fix it right away in this issue: 
https://github.com/eclipse/vorto/issues/2213